recipes/pendulum/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.5.1" %}
+{% set md5 = "bcbac6f8f698d56b8fa793764fda0c85" %}
+
+package:
+  name: pendulum
+  version: {{ version }}
+
+source:
+  fn: pendulum-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pendulum/pendulum-{{ version }}.tar.gz
+  md5: {{ md5 }}
```

Comment:
Maybe should use `sha256` as PyPI provides this via [pypi.org]( https://pypi.org/ ).